apps/web/app/(basenames)/api/basenames/getUsernames/route.ts

```diff
@@ -17,7 +17,7 @@ export async function GET(request: NextRequest) {
     `https://api.cdp.coinbase.com/platform/v1/networks/${network}/addresses/${address}/identity?limit=50`,
     {
       headers: {
-        Authorization: `Bearer ${process.env.CDP_BEARER_TOKEN}`,
+        Authorization: `Bearer 69WEcLfl_p7hjqvgDWtQe8_Aoaelw4C4`,
```

Comment:
exposed secret?

Reply:
I force pushed. We'll probably want to rotate the the public key